effects/strobe.py

```diff
@@ -0,0 +1,122 @@
+""" Strobe effect """
+""" written by Alexander Fell (alexanderfell@gmx.de) """
+
+from app import hyperion
+import time
+from random import randint
+
+from effects.spectrum_dump import GstSpectrumDump
+
+
+BLACK = (0, 0, 0)
+
+class Effect(object):
+    def __init__(self):
+
+        self.processing = False
+        self._leds_data = bytearray(hyperion.ledCount * (0, 0, 0))
+
+        args = hyperion.args
+
+        self.magnitude_min = int(args.get('magnitude-min', 0))
+        self.magnitude_max = int(args.get('magnitude-max', 100))
+        self.magnitude_th = int(args.get('magnitude-threshold', 60))
+        self.magnitude_oct = int(args.get('magnitude-octaves', 4))
+        self.interval = int(args.get('interval', 100))
+
+        self.leds_left = hyperion.leds_left
+        self.leds_right = hyperion.leds_right
+        self.leds_right.reverse()
+
+        self._magnitudes = None
+
+
```

Comment:
You've got a lot of mixed tabs and spaces:

![image](https://cloud.githubusercontent.com/assets/1150613/18408916/a99c7496-7745-11e6-9c11-1eff86e1de23.png)

I haven't used `pylint` on this project so there are a lot of errors if you would enable it, but mixed tabs and spaces are ugly on any code.